registry/registry_ui.go

```diff
@@ -497,12 +495,19 @@ func createUpdateNamespace(ctx *gin.Context, isUpdate bool) {
 		ns.AdminMetadata.UserID = user
 		// Overwrite status to Pending to filter malicious request
 		ns.AdminMetadata.Status = Pending
+		if inTopo {
+			ns.AdminMetadata.Description = "[ Attention: Prefix exists in OSDF topology ] " + ns.AdminMetadata.Description
```

Comment:
And here as well -- we should be explicit about whether the namespace exists or whether some super-space of the attempted registration exists